packages/crowdstrike/data_stream/fdr/elasticsearch/ingest_pipeline/default.yml

```
@@ -2618,12 +2627,17 @@ processors:
       field: crowdstrike.User.Name
       target_field: user.name
       ignore_missing: true
-      if: ctx.crowdstrike?.User?.Name != null && ctx.user?.name == null
+      if: '!(ctx.crowdstrike?.User instanceof String) && ctx.crowdstrike?.User?.Name != null && ctx.user?.name == null'
```

```suggestion
      if: 'ctx.crowdstrike?.User instanceof Map && ctx.crowdstrike?.User?.Name != null && ctx.user?.name == null'
```

If ctx.crowdstrike.User were an array or another type without a Name field, attempting to read .Name could fail.